### DISCUSSION TITLE
¿Pagina o Sitio web?

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#25] [Q&A] [answered]
Como su nombre lo indica, al hacer las prácticas aparece como descripción que es una página web. 

Pero al tener varias páginas (login, página de administrador, etc) no ¿será un sitio web?

No se si estoy equivocado o realmente todo se genera a través de una misma página web.

**Comment by AngelCOUACM** [comment, marked as answer]
Para eso es el análisis de requisitos, al ir levantando cada necesidad al final tendrás la capacidad de decidir que tipo de sistema es

**Reply by OmarVenturaP** [reply]
Gracias. De hecho fue lo primero que me di cuenta, nada más de leer la práctica. Soy novato en esto del desarrollo por ello pense que era algo diferente o que nos estaban poniendo una "trampita" alli.  jeje

**Reply by AngelCOUACM** [reply]
Conforme vayas practicando, las cosas se hacen cada vez más fáciles.

**Reply by OmarVenturaP** [reply]
Muchas gracias Angel. Saludos

**Comment by DavidMaralb** [comment]
Toma en cuenta que una persona fuera del área, identificara cualquier aplicación web como Pagina web, Por ejemplo en la redacción del documento de requerimientos, lo mas seguro es que el cliente solicite una pagina para la firma de abogados, pero nosotros como especialistas en el área, le demos el termino correcto "Aplicación web". Saludos
